client/modules/api/README.md

```diff
@@ -0,0 +1 @@
+# @wikijump/api
```

Comment:
Are all the modules README files blank like this?

Reply:
Anything that's been published has a detailed README. Others have nothing like this or just a tiny description. This module is so barebones right now I didn't bother to write a blurb. This will change as I add stuff to the frontend that makes use of the API.

Reply:
Fair enough.